helm/helm-2to3-migration/templates/cm.yaml

```diff
@@ -21,18 +21,35 @@ metadata:
 data:
   run.sh: |-
     #!/bin/sh
+
     i=0
+    j=0
+
     for r in $(cat /data/config/releases | tr "," "\n");
     do
-      if ! 2to3 convert --tiller-ns {{ .Values.tiller.namespace }} "$r";
-      then
-         i=$((i+1));
+      errFile=$(mktemp)
```

Comment:
I'd still like the job to output stdout but I couldn't see a way to do it without a temp file.

In the PSP `readOnlyRootFilesystem: false` so I hope that will be OK.

Reply:
If it works with temp file that's cool, no need to change it.

Just in case, there is a way to capture and print, you have to setup another file descriptor to redirect to stdout so you can use it for printing, and then you can capture FD 1/2, e.g.:

``` sh
# redirect FD 8 to stdout
exec 8>&1

# pipe to tee which outputs to stdout (so gets captured) and also to FD 8
# which is redirected to stdout (FD 1) so gets printed
captured="$(2to3 convert --tiller-ns {{ .Values.tiller.namespace }} "$r" 2>&1 | tee >(cat - >&8))"
```

Reply:
Thanks @mig4 I tried it without the temp file but couldn't get it working.
 
I'm going to go with the temp file but this is nice and I'll use it in future. 👍